examples/first-component/src/components/ChatRoom/index.tsx

```diff
@@ -0,0 +1,25 @@
+// https://react.dev/reference/react/useEffect#removing-unnecessary-object-dependencies
+
+import { useState } from "react";
+import Room from "../Room";
+
+const ChatRoom = () => {
+  const [roomId, setRoomId] = useState("general");
```

Comment:
I know this is example but u need to care more about type safety in this init state, "general" just a string it can mismatch with below options

Reply:
Fixed and I add another case is the empty value to check